js/style/style.js

```diff
@@ -731,7 +732,7 @@ Style.prototype = util.inherit(Evented, {
 
     // Callbacks from web workers
 
-    'get sprite json': function(params, callback) {
+    'get sprite json': function(_, params, callback) {
```

Comment:
For these targets on the main thread, this is a dummy parameter.  The only reason it exists here is so that `Actor` can be symmetric -- that is, Actor can consistently call the target method on its `parent` with (parentId, data, callback), even though parentId is irrelevant if it's receiving message from the worker for the main thread